### PR TITLE
refactor: 解决子包循环依赖问题

### PR DESCRIPTION
**Pull request by hexqi**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/http/src/environments.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const VITE_ORIGIN = import.meta.env.VITE_ORIGIN
+export const isMock = import.meta.env.VITE_API_MOCK === 'mock'
```
Comment (chilingling) [inline packages/http/src/environments.js]: `import.meta ` 是编译的时候注入的，这里编译的时候没有相关环境变量，注入会不正确

File: packages/common/index.js [hunk 2]
```diff
@@ -90,6 +90,20 @@ const { mergeLocaleMessage } = i18n.global
 mergeLocaleMessage(i18nKeyMaps.enUS, enUs)
 mergeLocaleMessage(i18nKeyMaps.zhCN, zhCn)
 
+const globalComponents = {
+  SaveNewBlock,
```
Comment (chilingling) [inline packages/common/index.js:94]: 关于  MetaComponent ，建议在 controller 中设置状态变量保存，通过类似  get、set  方法进行获取保存。这样可以：
1. 第三方注册自定义  MetaComponent 
2. 避免使用 global component，避免冲突

后续  MetaComponent 需要与 common package  分离

File: packages/http/src/environments.js (hunk 1 above)
Comment (hexqi) [reply packages/http/src/environments.js]: 已修改

File: packages/common/index.js (hunk 2 above)
Comment (hexqi) [reply packages/common/index.js:94]: 方案还需要讨论，后续components包重构时修改

**Review by chilingling — APPROVED** [review]

**Pull request merged**